packages/shared/src/components/plus/PlusListItem.tsx

```diff
@@ -0,0 +1,154 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import ConditionalWrapper from '../ConditionalWrapper';
+import { SimpleTooltip } from '../tooltips';
+import { ChecklistAIcon, InfoIcon } from '../icons';
+import { IconProps, IconSize } from '../Icon';
+import {
+  Typography,
+  TypographyColor,
+  TypographyProps,
+  TypographyTag,
+  TypographyType,
+} from '../typography/Typography';
+import { usePaymentContext } from '../../contexts/PaymentContext';
+
+export enum PlusItemStatus {
+  Ready = 'done',
+  Disabled = 'disabled',
+  ComingSoon = 'coming-soon',
+}
+
+export interface PlusItem {
+  label: string;
+  status: PlusItemStatus;
+  tooltip?: string;
+}
+
+export interface PlusListItemProps {
+  item: PlusItem;
+  typographyProps?: TypographyProps<TypographyTag.P>;
+  iconProps?: IconProps;
+  badgeProps?: TypographyProps<TypographyTag.Span>;
+}
+
+export const SimplePlusListItem = ({
+  item,
+  typographyProps,
+  iconProps,
+}: PlusListItemProps): ReactElement => {
+  return (
+    <ConditionalWrapper
+      condition={!!item.tooltip}
+      wrapper={(component: ReactElement) => (
+        <SimpleTooltip
+          container={{ className: 'tablet:max-w-72' }}
+          content={item.tooltip}
+          delay={0}
+        >
+          {component}
+        </SimpleTooltip>
+      )}
+    >
+      <li
+        className={classNames(
+          '-m-px flex items-center gap-2 rounded-6 p-1',
+          !!item.tooltip && 'hover:bg-surface-float',
+        )}
+      >
+        <ChecklistAIcon
+          size={IconSize.Small}
+          {...iconProps}
+          className={classNames('text-text-quaternary', iconProps?.className)}
+        />
+        <Typography
+          tag={TypographyTag.P}
+          type={TypographyType.Body}
+          color={TypographyColor.Primary}
+          {...typographyProps}
+          className={classNames('flex-1', typographyProps?.className)}
+        >
+          {item.label}
+        </Typography>
+        {item.tooltip && <InfoIcon />}
+      </li>
+    </ConditionalWrapper>
+  );
+};
+
+const PlusListItemWithComingSoon = ({
```

Comment:
These are not basically the same, one just adds something right?
Not urgent, but maybe once live we can clean it up a bit?